src/lib/kafkajs/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./JSConsumer";
+export * from "./JSProducer";
```

Comment:
minimal stuff, lots of files missing ending new line, you should probably change this in your editors settings :D

Reply:
🤔 i have a feeling that eslint --auto-fix removed them as i normally leave a new line. Will look into and fix up.

Reply:
Nope, it was actually my editor 🤭